LICENSE

```diff
@@ -1,21 +1,674 @@
-The MIT License (MIT)
-
-Copyright (c) 2017 Luc Didry
```

Comment:
Well legally speaking I don't know if we're allowed to just yolochange the license just like that but idk

Reply:
The upstream is GNU Affero General Public License v3.0. Let's put it back like it was.

Reply:
The upstream software is AGPLv3, but I choose to put the scripts of the package under MIT License.
As I’m the author of the upstream software and the original author of the YNH packaging, I think I can choose any license I want, so let’s keep it that way (AGPLv3 + MIT) :grin:

Reply:
You are absolutely right. It is as it was before.